t/ui/06-operator_links.t

```diff
@@ -35,59 +35,48 @@ unless ($driver) {
 my $t = Test::Mojo->new('OpenQA::WebAPI');
 
 # we don't want to test javascript here, so we just test the javascript code
-# List with no login
+note 'List with no login';
 $t->get_ok('/tests')->status_is(200)->content_like(qr/is_operator = false;/, "test list rendered without is_operator");
-
-# List with an authorized user
+note 'List with an authorized user';
 $test_case->login($t, 'percival');
 $t->get_ok('/tests')->status_is(200)->content_like(qr/is_operator = true;/, "test list rendered with is_operator");
-
-# List with a not authorized user
+note 'List with a not authorized user';
 $test_case->login($t, 'lancelot', email => 'lancelot@example.com');
 $t->get_ok('/tests')->status_is(200)->content_like(qr/is_operator = false;/, "test list rendered without is_operator");
-
-# now the same for scheduled jobs
+note 'now the same for scheduled jobs';
 $t->delete_ok('/logout')->status_is(302);
-
-# List with no login (absence of cancel button already checked in 01-list.t)
+note 'List with no login (absence of cancel button already checked in 01-list.t)';
 $t->get_ok('/tests')->status_is(200);
-
-# List with an authorized user (presence of cancel button already checked in 01-list.t)
+note 'List with an authorized user (presence of cancel button already checked in 01-list.t)';
 $test_case->login($t, 'percival');
 $t->get_ok('/tests')->status_is(200);
-
-# List with a not authorized user
+note 'List with a not authorized user';
 $test_case->login($t, 'lancelot', email => 'lancelot@example.com');
 $t->get_ok('/tests')->status_is(200);
-
-# operator has access to part of admin menu - using phantomjs
+note 'operator has access to part of admin menu';
 $driver->title_is("openQA", "on main page");
 $driver->find_element_by_link_text('Login')->click();
-# we're back on the main page
+note 'we are back on the main page';
 $driver->title_is("openQA", "back on main page");
-# but ...
-
+note 'but ...';
 is($driver->find_element('#user-action a')->get_text(), 'Logged in as Demo', "logged in as demo");
-
-# now hack ourselves to be just operator - this is stupid procedure, but we don't have API for user management
+note 'now hack ourselves to be just operator - this is stupid procedure, but we do not have API for user management';
 $driver->find_element('#user-action a')->click();
 $driver->find_element_by_link_text('Users')->click;
 $driver->execute_script('$("#users").off("change")');
 $driver->execute_script(
     '$("#users").on("change", "input[name=\"role\"]:radio", function() {$(this).parent("form").submit();})');
 $driver->find_element_by_xpath('//tr[./td[@class="nick" and text()="Demo"]]/td[@class="role"]//label[2]')->click;
-
-# refresh and return to admin pages
+note 'refresh and return to admin pages';
 $driver->refresh;
 $driver->get($driver->get_current_url =~ s/users//r);
 
 $driver->find_element('#user-action a')->click();
-# we should see test templates, groups, machines
+note 'we should see test templates, groups, machines';
```

Comment:
Maybe some of these notes should be subtests instead, e.g. `subtest 'test templates, groups and machines shown' => `.

Reply:
I prefer to not introduce subtests here as the test module basically still follows one story from top to bottom, not multiple ones.